Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml

```
@@ -18,69 +18,53 @@ args:
   name: brands
   auto: PREDEFINED
   predefined:
-  - 'SailPointIdentityNow'
   - 'Active Directory Query v2'
-  - 'PingOne'
   - 'Okta v2'
   - 'AWS - IAM'
   - 'Microsoft Graph User'
-  - 'SailPointIdentityIQ'
   - 'Cortex XDR - IR'
   - 'Cortex Core - IR'
-  - 'iam-get-user'
+  - 'PrismaCloud v2'
+  - 'AzureRiskyUsers'
 - description: Whether to retrieve human readable entry for every command or only the final result. True means to retrieve human readable entry for every command. False means to human readable only for the final result.
   name: verbose
   auto: PREDEFINED
   predefined:
   - 'true'
   - 'false'
+- description: whether there are additional fields to the main fields or not.
```

```suggestion
- description: Whether to return unmapped fields to the context output under the "AdditionalFields" path. 
```

according to the standard